spec/manageiq-operator/crd_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'awesome_spawn'
+describe "Generate CRDs" do
+  it "doesn't change any tracked files" do
+    AwesomeSpawn.run!("operator-sdk generate k8s", :chdir => ROOT.join("manageiq-operator"))
+    AwesomeSpawn.run!("operator-sdk generate crds", :chdir => ROOT.join("manageiq-operator"))
+    AwesomeSpawn.run!("operator-sdk generate csv --update-crds --csv-version=0.0.1", :chdir => ROOT.join("manageiq-operator"))
```

Comment:
This version string might be problematic once we start actually releasing versions, but I don't think we're there yet. Just something to consider.

Reply:
Is `manageiq-operator/version/version.go:4: Version = "0.0.1"` a safe place to get the version from?

Reply:
I'm not actually sure. The operator might be versioned separately from the csv, but I haven't learned enough about either yet to say for sure. My guess would be that they would be the same version, but I also don't love the idea of parsing the version out of source to run this test.

I would say that this is fine for now.

Reply:
> The CSV version is the same as the Operator’s, and a new CSV is generated when upgrading Operator versions.

ref: https://docs.openshift.com/container-platform/4.4/operators/operator_sdk/osdk-generating-csvs.html

Reply:
Done.